### DISCUSSION TITLE
Milestone 1 feedback

### DISCUSSION THREAD

**Discussion by GBKS** [BitcoinDesign/Bitcoin-Core-App#45] [General]
Here you can share your feedback on the milestone 1 release of the Bitcoin Core App. Before commenting, please review if your topic has already been addressed.

**Comment by epiccurious** [comment]
I did a full sync on mainnet and have some feedback. Didn't see these discussed topics in the open/closed issues and PRs but let me know if I missed a conversation.

Tested on a Google Pixel 6a running Android 13 firmware. I used the ARM64 Android apk from the `Checks` of [this pull request](https://github.com/jarolrod/gui-qml/pull/5), Version v.24.99.0-e0aacd194c3a.

Feature requests:
1. Add an option to see the equivalent of `tail -f .bitcoin/debug.log` and/or an option to scroll the entire debug file under Developer options.
2. Ability to edit `bitcoin.conf` or add custom arguments under the Developer option.
3. Add support for I2P.
4. Add support for CJDNS.
5. Ability to view more detailed IBD stats, like synced chain height, header chain height, and time of synced chain. Might be appropriate as an option such as "View more sync details" when tapping on the horizontal dots in the block clock.
6. Add a Settings option to toggle showing the amount of data sent/received for each peer to the Peers screen.
7. A step during the "first run" process of the app to guide the user about managing system sleep and battery.
8. Add a page in the About section of settings explaining what the block clock visualized. The app never explains what each part of the block clock represents. Why is some of it yellow and some red sometimes and completely red at other times? What are the black ticks on the ring? What does it mean when the block clock is filled?

UI/Design suggestions:
1. The "`Showcase` Block Clock display mode for is more appropriate on a Pixel 6a (6.1", 1080x2400 screen). The `Default` display mode seems too small. Will post four screenshots below showing the two display modes below in both portrait and landscape orientations.
2. On the Peers screen, the text on the buttons at the top to sort (ID, Direction, User Agent, Type, ...) are smaller than the text for the peers. With a 6.1" or larger screen, it might make sense to scale up the buttons a bit so the text is the same size.
3. On the Peers screen, when connected to 10 peers, scrolling to the bottom of the list gets rid of the sort buttons. Instead, keep the sort buttons on the screen and have the first few peers scroll off the screen.
4. Add a smooth effect to the network charts, especially the 5-minute chart. Maybe smoothed by default with a settings option to plot the raw data.
5. When you go `back` from the main screen to shut down the app, the display says "Saving... Do not shut down the computer until it is done...'. This doesn't seem appropriate for an Android app. Replace "the computer" with "your device". Also replace "this is done" with "this process completes" to improve grammar.
6. During the "first run" process of the app, change the label from `Next` to `Start` on the last page (on the page saying Starting initial download...) to indicate to the user that this is their last opportunity to go "Back".
10. During the "first run" process of the app, the page headings bounce around. Medium on the 1st page, low on the 2nd, 3rd, and high on the 4th and 5th pages. The text on first page (saying "Bitcoin Core App") could be brought down a bit to be in line with the heading on the second page (saying "Strengthen bitcoin") and 
7. During the "first run" process, block clock is in lowercase. In the display settings, Block Clock is in title case. One of these should be changed to keep consistency in capitalization.

I'd be happy to create issues for each of these, if appropriate, but wanted to bring them up here first.

**Comment by epiccurious** [comment]
Posting screenshots related to for UI/Design suggestion #1. Taken on a Pixel 6a (6.1", 1080x2400 screen).

Default display mode in portrait
![Default display mode in portrait](https://github.com/BitcoinDesign/Bitcoin-Core-App/assets/109078515/eb587b93-9232-433c-bb01-fcae4b6af5c6)

Showcase display mode in portrait
![image](https://github.com/BitcoinDesign/Bitcoin-Core-App/assets/109078515/f7769c6a-bfca-4ea0-b534-ec3f13dfb29f)

Default display mode in landscape
![Default display mode in landscape](https://github.com/BitcoinDesign/Bitcoin-Core-App/assets/109078515/e1a33ca6-c2f1-4285-b61a-d9afa71c9aa1)

Showcase display mode in landscape
![Showcase display mode in landscape](https://github.com/BitcoinDesign/Bitcoin-Core-App/assets/109078515/b5900698-cc7e-4ddf-81ab-6ee6afc81cb4)

**Reply by princedede** [reply]
Please, where is the link to download??

**Comment by princedede** [comment]
I am looking for the download link guys. Thanks!

**Reply by GBKS** [reply]
Here you go. https://bitcoincore.app

**Comment by SatoshiNakamotoBitcoin** [comment]
Hi, we like it.
If is possible make an iOS TestFlight app or 1 .ipa can be amazing.

Congratulations.

We found this in old Os-

cannot load 'bitcoin-qt' (load command 0x80000034 is unknown)

10.xx.x intel.

**Reply by GBKS** [reply]
Sorry, no one has worked on getting the app working on iOS. We'd need some new contributors there with that skillset.

**Comment by pjw65** [comment]
Did you install it on a phone, tablet or desktop? Phone
What is the device brand and model? Xiaomi Mi Note 10 Lite
Which operating system did you use? Android 12
Were the download instructions easy to understand? yes

How was the initial installation process? 
For me it was ok, for a non programmer it would be difficult. 
From https://github.com/bitcoin-core/gui-qml/actions/runs/7518999590 (last updated 2 months ago) 
I did not know, which download I need (arm 64). 
I had to Unpack to a directory, than open the apk.
Then I was asked to change the settings to allow "downloaded" files to install, then it says it can't install, because the APK is not using
the most resent security needed, but there was a small link called "install anyway" (and 2 buttons to cancel / ok).
The installation after that was easy except the question about the "Storige Amount". I could choose betweend Reduced 8GB and Full 8GB. I selected "Reduced".

How long did it take to sync? 40min over WLAN (I like the Settings/Network traffic)
How did the installation affect the battery of your device? normal
What do you think of the block clock? nice, simple version of https://timechaincalendar.com
How was the overall experience for you? good

I also changed on the phone settings not to use "Energy saving".
Still "Signet Network"
Sync done at Block 188.192

**Reply by GBKS** [reply]
Thank you for the feedback. Installation is a bit of a hassle right now, but that's also because it's in-development code. For end-users, we'll hopefully have a smoother journey eventually.
